backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -114,9 +110,8 @@ public class UserController {
   public ResponseEntity<UserResponse> registerUser(
       @Valid @RequestBody UserRegisterRequest userRegisterRequest) {
 
-    UserDto userDto = userService.registerUser(modelMapper.map(userRegisterRequest, UserDto.class));
-
-    return new ResponseEntity<>(modelMapper.map(userDto, UserResponse.class), HttpStatus.OK);
+    ResponseEntity<UserResponse> responseEntity = new ResponseEntity<>(userService.registerUser(userRegisterRequest), HttpStatus.OK);
+    return responseEntity;
```

Comment:
Lines 113 and 114 could be one line

Reply:
Yes that's why i was doing this task, yes i guess it could be one line, but this way it's more consistent with the type of code we are writing imo.